pyccel/ast/numpyext.py

```diff
@@ -1201,8 +1224,38 @@ def __init__(self, x):
         self._precision = default_precision[str_dtype(self._dtype)]
 
 class NumpyMod(NumpyUfuncBinary):
-    def __init__(self, x1, x2):
-        self.copy_attributes(x1)
+    def __init__(self, *args):
```

Comment:
I think we should immediately raise an error if we pass the wrong number of arguments, so I prefer the original signature. The rest of the `__init__` function could remain the same, just write
```
def __init__(self, x1, x2):
    args = (x1, x2)
    ...
```
as you did for class `Matmul`

Reply:
yes you are right